main.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python2
+# encoding=utf8
+import sys
+import psutil
+import os
+import pprint
+from Core.Logger import logger
+from Core.common_functions import call_subprocess as sh
+from Core.common_functions import parse_script_commandline_arguments as parse_args
+from Core.SSH import Host
+
+FILES_TO_CLEATE = 10
```

Comment:
`FILES_TO_CLEATE` -- need to be a program argument, i.e. use `ArgumentParser` for this:
> create Z files of size Y

Reply:
refactored